chat-plugins/info.js

```diff
@@ -109,6 +106,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (user.can('mute', targetUser, room) && targetUser.userid in room.users)) {
+			var bannedFrom = "";
+			for (var i = 0; i < Rooms.global.chatRooms.length; i++) {
+				var thisRoom = Rooms.global.chatRooms[i];
+				if (!thisRoom || thisRoom.isPrivate) continue;
+				if (thisRoom.bannedIps && (targetUser.latestIp in thisRoom.bannedIps || targetUser.userid in thisRoom.bannedUsers)) {
```

Comment:
I think thats a useful information even for drivers. If a user is banned in many rooms, it's "obvious" a troll